.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,99 @@
+name: Bug Report
+description: File a bug report to help improve qBittorrent user experience.
+body:
+  - type: markdown
+    attributes:
+     value: |
+             #### ADVISORY
+             We **DO NOT** support any **End-Of-Life** versions `<=4.1.x` or **3rd party/forked versions** e.g. `portableapps`/`Enhanced Edition`etc.
```

Comment:
Too many bold text and full uppercase words makes it hard to read (it looks like written by an edgy kid and put me off), please refrain from using it or use it scarcely.

The same applies to other labels.

Reply:
About the EOL versions, I would state it as:
"We do not support any versions older than the current release series"
This is to avoid mentioning exact numbers which calls for regular maintenance for it.

Reply:
>or **3rd party/forked versions** e.g. `portableapps`/`Enhanced Edition`etc.

Any issues with this? (un-bolded)

Reply:
>Too many bold text and full uppercase words makes it hard to read (it looks like written by an edgy kid and put me off), please refrain from using it or use it scarcely.

The same applies to other labels.

Most of the bold text words are just a copy&paste of our current template